Doc/library/ast.rst

```diff
@@ -1820,7 +1820,7 @@ Function and class definitions
             type_ignores=[])
 
 
-.. class:: ClassDef(name, bases, keywords, starargs, kwargs, body, decorator_list)
+.. class:: ClassDef(name, bases, keywords, body, decorator_list)
```

Comment:
This is also now out of date with the implementation in main and 3.12 due to the addition of `type_params` argument in PEP 695, but that's being addressed in https://github.com/python/cpython/pull/104642

Still seems sensible to remove `starargs` and `kwargs` in this PR, and let @JelleZijlstra resolve the resulting conflict on this line in https://github.com/python/cpython/pull/104642

Reply:
Hmm, backporting this will mean that the PEP 695 docs PR won't backport cleanly, which may cause a bit more of a pain. @JelleZijlstra how do you prefer to handle this?

Reply:
Just merge this first, I'll take care of the backports